f5_heat/resources/f5_bigip_device.py

```diff
@@ -52,6 +52,7 @@ class F5BigIPDevice(resource.Resource):
 
     def __init__(self, name, definition, stack):
         super(F5BigIPDevice, self).__init__(name, definition, stack)
+        self.bigip = None
```

Comment:
I don't understand what this assignment statement does.  I think it should probably be removed.